### PR TITLE
added children column to main registration table

### PR DESCRIPTION
**Pull request by Lambo-Liu**
### Notion ticket link

["Children" column](https://www.notion.so/uwblueprintexecs/9cb454262ed84c6299c0682c23f92ec5?v=1076bd30711a4a01838becee05f8b5a3&p=d159dadec85f44bd9aba8745a24b6105)

### Implementation description

- added a "children" column to the main registration table, displaying the first name and age of each child in the family.

### Steps to test

1. start the frontend and look at the registration table :)

### What should reviewers focus on?

- clean code, typescript conventions, etc.

### Checklist

- [x] My PR name is descriptive and in imperative tense
- [x] I have run the linter
- [x] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by Tayef-Shah** [comment]
@Lambo-Liu Just a couple of suggestions for the PR description
- Renaming the link to the ticket name with the actual ticket name is helpful
> !!! WAIT FOR "add-children-column" BACKEND TO MERGE FIRST !!!
- ^Linking the pull request for that branch would be sick, cuz I don't know branches by name lol

**Comment by Lambo-Liu** [comment]
> @Lambo-Liu Just a couple of suggestions for the PR description
> 
> * Renaming the link to the ticket name with the actual ticket name is helpful
> 
> > !!! WAIT FOR "add-children-column" BACKEND TO MERGE FIRST !!!
> 
> * ^Linking the pull request for that branch would be sick, cuz I don't know branches by name lol

Forgot about this! Thanks!

File: src/constants/DefaultFieldKey.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ enum DefaultFieldKey {
   ID = "id",
   LAST_NAME = "last_name",
   NUM_CHILDREN = "num_children",
+  CHILDREN_INFO = "children",
```
Comment (brittanylau) [inline src/constants/DefaultFieldKey.ts:12]: nit: if `CHILDREN` works can we use that instead of `CHILDREN_INFO` 🙈

**Review by brittanylau — APPROVED** [review]
logic is squeaky clean 🛁  just some smol nits

**Pull request merged**